layers/+spacemacs/spacemacs-defaults/packages.el

```diff
@@ -407,7 +407,9 @@
       (add-to-list 'recentf-exclude
                    (recentf-expand-file-name spacemacs-cache-directory))
       (add-to-list 'recentf-exclude (recentf-expand-file-name package-user-dir))
-      (add-to-list 'recentf-exclude "COMMIT_EDITMSG\\'"))))
+      (add-to-list 'recentf-exclude "COMMIT_EDITMSG\\'")
+      (unless (null custom-file)
```

Comment:
you can replace `(unless (null ..) ..)` by `(when ..)`

Reply:
I have fixed this during the merge